### PR TITLE
fix(ingest/tableau): handle missing query in tableau views

### PR DESCRIPTION
**Pull request by hsheth2**
Fixes https://github.com/datahub-project/datahub/issues/7182

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Comment by khgould** [comment]
Darn you beat me to it. I came here to submit the easy PR. You're amazing @hsheth2 !

File: metadata-ingestion/src/datahub/ingestion/source/tableau.py [hunk 1]
```diff
@@ -893,7 +893,7 @@ def emit_custom_sql_datasources(self) -> Iterable[MetadataWorkUnit]:
             view_properties = ViewPropertiesClass(
                 materialized=False,
                 viewLanguage="SQL",
-                viewLogic=clean_query(csql.get("query", "")),
+                viewLogic=clean_query(csql.get("query") or ""),
```
Comment (mayurinehate) [inline metadata-ingestion/src/datahub/ingestion/source/tableau.py:896]: Suggestion: Let's just skip adding viewProperties to dataset aspects if custom SQL query is not available.

**Comment by khgould** [comment]
Hey  -  can we get this merged 🙏  
Either solutions works.

**Review by mayurinehate — APPROVED** [review]
LGTM 
One small suggestion but that can be added in a followup PR

**Review by jjoyce0510 — APPROVED** [review]

**Pull request merged**